packages/marshal/src/marshal.js

```diff
@@ -1089,3 +1132,16 @@ const Far = (farName, remotable = {}) =>
 
 harden(Far);
 export { Far };
+
+function Data(props) {
+  assert(
+    ownKeys(props).length === 0 || isPassByCopyRecord(props),
+    X`Data() can only be applied to otherwise pass-by-copy records`,
+  );
+  Object.defineProperty(props, PASS_STYLE, {
+    enumerable: false,
+    value: 'copyRecord',
+  });
+  return harden(props);
+}
+export { Data };
```

Comment:
Follow the pattern for other exported functions.

```suggestion
harden(Data);
export { Data };
```